tests/foreman/cli/test_import.py

```diff
@@ -78,52 +80,64 @@ def update_csv_values(csv_file, new_data=None, dirname=None):
     :param csv_file: A string. The path to a CSV file that resides
         on a remote server.
     :param new_data: A list of dictionary objects. Each object has to
-        contain an org_id key corresponding to organization_id of the record
-        being updated and the key-value pairs of attributes to be changed for
-        the particular records. For example::
+        contain an key_id key corresponding to org[anization]_id of the
+        record being updated and the key-value pairs of attributes to be
+        replaced by the particular records. For example::
 
             [
                 {
-                    u'org_id': u'1',
+                    u'key_id': u'1',
                     u'organization': u'updated_organization_name_1',
                     u'username': u'updated_user_name_1',
                 },
                 {
-                    u'org_id': u'2',
+                    u'key_id': u'2',
                     u'organization': u'updated_organization_name_2',
                     u'username': u'updated_user_name_2',
                 }
             ]
 
-    :returns: A string. The path to a locally created CSV file
+    :returns: A string. The path to a remotely created CSV file
 
     """
+    updated = False
     # if new_data is not specified, no change happens
     if new_data is None:
         return csv_file
-
     result = csv_to_dataset([csv_file])
     for change in new_data:
+        if change.get('key_id') is None:
+            continue
         for record in result:
-            if record['organization_id'] == change['organization_id']:
+            if (record.get('organization_id') == change['key_id'] or
+                    record.get('org_id') == change['key_id']):
                 record.update(change)
-    return build_csv_file(result, dirname)
+                del record['key_id']
+                updated = True
+    if updated:
+        return build_csv_file(result, dirname)
+    else:
+        return csv_file
 
 
 def import_org_data():
     """Random data for Organization Import tests"""
 
+    random_ids = random.sample(range(1, 1000), 3)
     return ([
```

Comment:
Or you can go even further, making `import_org_data()` more generic, adding optional arg, e.g.
```python
 def import_org_data(count=3):
    """Random data for User Import tests"""
    random_ids = sample(range(1, 1000), count)
        return [
            {
                u'key_id': str(index+1),
                u'organization_id': str(random_id),
                u'organization': gen_string('alphanumeric'),
                u'username': gen_string('alphanumeric')
            }
            for index, random_id
            in enumerate(random_ids)
        ]
```
So there could be tests with random amount of data, not just with predefined 3 items.
BTW, why do you need a tuple of 1 element?